tests/Integration/Behaviour/Features/Context/Domain/CartFeatureContext.php

```diff
@@ -339,7 +339,7 @@ public function checkCartCarrier(string $cartReference, string $carrierReference
     public function setFreeShippingToCart(string $cartReference)
     {
         $this->getCommandBus()->handle(
-            new SetFreeShippingToCartCommand(
+            new UpdateCartDeliverySettingsCommand(
```

Comment:
Could you add a behat test to ensure the command correctly handles these two new fields?